semgrep_output_v1.atd

```diff
@@ -2023,7 +2023,7 @@ type analyzer <ocaml attr="deriving show"> = string wrap <ocaml module="Analyzer
 *)
 type target <ocaml attr="deriving show"> = [
```

Comment:
We could also change the targets type and have there a list of code_target and a list of dependency_source.
But let's go with this approach.
Note that @mjambon added also a new Scanning_roots cases in the targets type where pysemgrep relies on
osemgrep file targeting to compute the targets, but I think we're fine with your change; just in the futur for those
cases where pass the Scanning_roots, it will also be the job of the OCaml code to compute the dependency_source (using @bkettle subproject matcher probably).